lib/src/mt_rss.c

```diff
@@ -88,6 +88,8 @@ static int rss_flow_check(struct mtl_main_impl* impl, enum mtl_port port,
   enum mt_rss_mode sys_rss_mode = mt_get_rss_mode(impl, port);
   enum mt_rss_mode flow_rss_mode = rss_flow_mode(flow);
 
+  if (mt_if(impl, port)->drv_type == MT_DRV_ENA) return 0;
```

Comment:
Maybe below WA is more better. l3_l4 works if source port is same to dst port for tx stream, our TX is exactly this case.
```
if (sys_rss_mode == flow_rss_mode) return 0;
if (sys_rss_mode == l3_l4 && flow_rss_mode == l3_l4_dst_port_only) {
  warn(".... require l3_l4_dst_port_only but only l3_l4 allowed");
  return 0;
}
```